Duffel.ApiClient/Resources/OfferRequests.cs

```diff
@@ -10,7 +10,27 @@
 
 namespace Duffel.ApiClient.Resources
 {
-    public class OfferRequests : BaseResource<OffersResponse>
+    public interface IOfferRequests
+    {
+        /// <summary>
+        /// 
```

Comment:
```suggestion
        /// To search for flights, you'll need to create an offer request. An offer request describes the passengers and where and when they want to travel (in the form of a list of slices). It may also include additional filters (e.g. a particular cabin to travel in).
```